apps/api/src/app/workflows-v2/usecases/preview/preview.usecase.ts

```diff
@@ -306,6 +333,56 @@ export class PreviewUsecase {
     }
   }
 }
+
+/**
+ * Clean the preview payload example before returning to remove digest eventCount, events.length and ensure events array exists
+ */
+function cleanPreviewExamplePayload(payloadExample: Record<string, unknown>): Record<string, unknown> {
```

Comment:
Nit-picky comment: As per Lexicon, this is the [Preview Control Data or Preview Data](https://www.notion.so/novuhq/Product-Terminology-Lexicon-d6ef923953744bc1b967ba8f297dee7f?pvs=4#15ad03ad838f80cbb958e9906eddee46). 

Should we update the function name or the Lexicon?